www/src/utils/object.ts

```diff
@@ -0,0 +1,5 @@
+function IsObjectEmpty<K extends keyof any = string | number, T = unknown>(obj: {[P in K]?: T} | undefined) {
```

Comment:
https://lodash.com/docs/4.17.15#isEmpty

Reply:
This logic is different from the lodash `isEmpty`.

From the lodash docs:
> Objects are considered empty if they have no own enumerable string keyed properties.

Here, the object is considered empty not only if it does not have any keys, but also if all existing keys do not have values.

Reply:
Ok, makes sense. If we want to use Lodash, you could still use https://lodash.com/docs/4.17.15#omitBy and `isEmpty` together. Not sure if it makes sense though.